Dockerfile

```diff
@@ -1,19 +1,34 @@
-FROM codeclimate/alpine-ruby:b38
+FROM alpine:3.6
```

Comment:
What do you think of using the `2.4-alpine3.6` base image so we can skip the ruby/bundler install?

Reply:
For whatever reason I had issues with that base image. Engine code could not find bundler-audit files.
```
`require': cannot load such file -- bundler/audit/scanner (LoadError)
```

I spent more time than I should've had on it and didn't find the cause of it. Vanilla Alpine and packaged Ruby worked so I stuck with it.